Kitodo-DataManagement/pom.xml

```diff
@@ -162,6 +162,12 @@
             <artifactId>jandex</artifactId>
             <version>2.0.3.Final</version>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <version>3.1.0</version>
```

Comment:
Please use for version a global property as this dependency is used in `Kitodo/pom.xml` too